beartype_test/a00_unit/a90_decor/a60_main/test_codemain.py

```diff
@@ -12,13 +12,22 @@
 '''
 
 # ....................{ IMPORTS                           }....................
+import re
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 # WARNING: To raise human-readable test errors, avoid importing from
 # package-specific submodules at module scope.
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 from beartype.roar import BeartypeDecorHintPep585DeprecationWarning
 from beartype_test._util.mark.pytmark import ignore_warnings
 
+
+def _strip_ansi_escape_sequences(text: str) -> str:
+    '''
+    Strip all ANSI escape sequences from the passed string.
+    '''
+    return re.sub(r'\\033[^m]*m', '', text)
```

Comment:
@leycec i realized I may have made a mistake here. Should it be `\033` without the extra `\`? Or does it need to be in hex format? When I did this I thought I needed to match the literal `\033`. Sorry this really is my first time working with escape sequences. I can run a test today. 

The tests passed because the colors were not added in the environment so there's nothing to match for.

Reply:
Looks like either way works 🤔

Reply:
**Fascinating.** You're... right. I'm not quite sure, honestly. The intersection of ANSII escape sequences with raw regex syntax is surprisingly non-trivial. Thanks a heap for investigating that!

Luckily for us, it looks like you already nailed it on the first try. Just in case something actually *is* awry here, I've added an internal `FIXME:` comment to that line suggesting further research.

As always, you amaze. :bow: